luaui/Widgets/Tests/cmd_blueprint/test_cmd_blueprint_filter.lua

```diff
@@ -0,0 +1,109 @@
+local widgetName = "Blueprint"
+
+function skip()
+	return Spring.GetGameFrame() <= 0
+end
+
+function setup()
+	assert(widgetHandler.knownWidgets[widgetName] ~= nil)
+
+	Test.clearMap()
+
+	initialWidgetActive = widgetHandler.knownWidgets[widgetName].active
+	if initialWidgetActive then
+		widgetHandler:DisableWidget(widgetName)
+	end
+	widgetHandler:EnableWidget(widgetName, true)
```

Comment:
You can now replace this by Test.prepareWidget(widgetName). Also need to rebase or merge master to fix the same lines on test_cmd_blueprint_line.lua and test_cmd_blueprint_single.lua, they're already adapted in master.